Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build the fuseml_core binary
+FROM golang:1.16 as builder
+
+WORKDIR /workspace
+
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY gen/ gen/
```

Comment:
Unfortunately for directories `COPY` behaves a bit different. If I try to copy multiple directories it will copy only the contents of the directory instead of the directory itself.